### PR TITLE
feat: improve codespace init time & experience

### PR DESCRIPTION
**Pull request by darmalovan**
# Description

Improvements to make docker compose run faster.

### CODE REVIEW

File: .devcontainer/setup.sh [hunk 1]
```diff
@@ -15,4 +15,9 @@ echo "export CODESPACE_WDS_SOCKET_PORT=443" >> ~/.bashrc
 echo "printf \"\n\n☁️☁️☁️️ Anythink: Develop in the Cloud ☁️☁️☁️\n\"" >> ~/.bashrc
 echo "printf \"\n\x1b[31m \x1b[1m👉 Type: \\\`docker compose up\\\` to run the project. 👈\n\n\"" >> ~/.bashrc
 
-nohup bash -c "cd /wilco-agent && node agent.js &" >> /tmp/agent.log 2>&1 
+nohup bash -c "cd /wilco-agent && node agent.js &" >> /tmp/agent.log 2>&1
+
+if command -v docker-compose &> /dev/null
```
[the remark this answers is not in the document]
Comment (darmalovan) [reply .devcontainer/setup.sh:20]: if docker compose exists

**Comment by shem8** [comment]
Remove the Docker file from each directory

**Comment by shem8** [comment]
Remove the Dockerfile files, now that we don't need it

**Review by shem8 — APPROVED** [review]

**Comment by darmalovan** [comment]
> Remove the Dockerfile files, now that we don't need it

@shem8 removed

**Pull request merged**